src/components/Molecules/CtaMessage/CtaMessage.css

```diff
@@ -0,0 +1,115 @@
+/* import colors... */
+@value colors: "../../00_global/colors.css";
+@value gray, orange, grayLighter from colors;
+
+/* import breakpoints */
+@value small as bp-small, medium as bp-medium from "../../00_global/breakpoints.css";
+
+@define-mixin prompt {
+  display: block;
+  position: relative;
+  box-sizing: border-box;
+  background-color: gray;
+  color: grayLighter;
+  text-align: center;
+}
+
+.prompt,
+.pushDown {
+  @mixin prompt;
+
+  padding: 35px 55px;
+}
+
+.loadUnder {
+  @mixin prompt;
+
+  position: fixed;
+  bottom: 0;
+  width: 100vw;
+
+  padding: 12px 42px;
+  border-top: 2px solid orange;
+}
+
+.close {
+  display: block;
+  position: absolute;
+  top: 0;
+  right: 0;
+
+  padding: 10px;
+  background: none;
+  border: none;
+
+  color: currentColor;
+
+  cursor: pointer;
+}
+
+.close:hover {
+  color: orange;
+}
+
+.logo {
+  margin-bottom: 1rem;
+}
+
+.title {
+  margin-bottom: 1rem;
+  color: inherit;
+  font-size: 1.5rem;
+}
+
+.description {
+  font-size: 1.1rem;
+  line-height: 2rem;
+}
+
+.actions * + * {
+  margin-left: 1rem;
+}
+
+.loadUnder .content > * {
+  display: inline-block;
+  margin-bottom: 0;
+}
+
+.loadUnder .content * + * {
+  margin-left: 1em;
+}
```

Comment:
I'd like to avoid using margin-left here. I'd suggest putting a margin on the logo itself. In smaller screen sizes the logo may go away.